docker-compose.production.yml

```
@@ -22,6 +22,27 @@ services:
     ports:
       - 8080:8080
 
+  admin:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.admin
+    restart: always
+    depends_on:
+      - postgres
+    env_file:
+      - backend/.env.docker
+    environment:
+      - ADMIN_PORT=8081
```

Shouldn't the environment be updated also to have ADMIN_COOKIE_SECRET?
```suggestion
    environment:
      - ADMIN_PORT=8081
      - ADMIN_COOKIE_SECRET=adminCookieSecret
```

---

Well this variable should be a secret so I wouldn't put it in the docker-compose file. To be frank I think `JWT_SECRET` should go as well from here, so that we don't set it to a default value which could be potentially dangerous

---

I agree. Shall we do it right away from this PR?